src/main/extension/extension_helper.cpp

```diff
@@ -123,6 +123,7 @@ static const DefaultExtension internal_extensions[] = {
     {"delta", "Adds support for Delta Lake", false},
     {"fts", "Adds support for Full-Text Search Indexes", false},
     {"ui", "Adds local UI for DuckDB", false},
+    {"ducklake", "Adds support for DuckLake", false},
```

Comment:
This description is different than the one registered in ducklake. I would keep the same in both places.